pkg/platform/kube/test/platform_test.go

```diff
@@ -1566,6 +1566,86 @@ def handler(context, event):
 	})
 }
 
+func (suite *DeployFunctionTestSuite) TestRedeployWithUpdatedSidecarSpec() {
+	functionName := "func-with-sidecar"
+	createFunctionOptions := suite.CompileCreateFunctionOptions(functionName)
+
+	sidecarContainerName := "sidecar-test"
+	commands := []string{
+		"sh",
+		"-c",
+		"for i in {1..10}; do echo $i; sleep 10; done; echo 'Done'",
+	}
+	newCommands := []string{
+		"/bin/sh",
+		"-c",
+		"for i in {1..5}; do echo $i; sleep 5; done; echo 'Done'",
+	}
```

Comment:
Let's change output to different strings, so we can check logs in both deployments and ensure that commands were applied correctly
```suggestion
	commands := []string{
		"sh",
		"-c",
		"for i in {1..10}; do echo $i; sleep 10; done; echo 'First deploy is done'",
	}
	newCommands := []string{
		"/bin/sh",
		"-c",
		"for i in {1..5}; do echo $i; sleep 5; done; echo 'Second deploy is done'",
	}
```